topoSort/ReconBuilder.py

```diff
@@ -0,0 +1,272 @@
+# ReconBuilder.py
```

Comment:
If you have time, please convert most of the dicts and trees to classes.

Reply:
will add to future tasks